CS3230Project/Model/Users/Patients/Patient.cs

```diff
@@ -107,101 +45,107 @@ public class Patient
             if (dateOfBirth > DateTime.Now)
-            {
                 throw new ArgumentException(PatientErrorMessages.DateOfBirthCannotBeInTheFuture);
-            }
-            if (gender == null)
-            {
-                throw new ArgumentException(PatientErrorMessages.GenderCannotBeNull);
-            }
-            if (gender.Trim().Length == 0)
-            {
-                throw new ArgumentException(PatientErrorMessages.GenderCannotBeEmpty);
-            }
-            if (addressOne == null)
-            {
-                throw new ArgumentException(PatientErrorMessages.AddressOneCannotBeNull);
-            }
+            if (gender == null) throw new ArgumentException(PatientErrorMessages.GenderCannotBeNull);
+            if (gender.Trim().Length == 0) throw new ArgumentException(PatientErrorMessages.GenderCannotBeEmpty);
+            if (addressOne == null) throw new ArgumentException(PatientErrorMessages.AddressOneCannotBeNull);
             if (addressOne.Trim().Length == 0)
-            {
                 throw new ArgumentException(PatientErrorMessages.AddressOneCannotBeEmpty);
-            }
-            if (city == null)
-            {
-                throw new ArgumentException(PatientErrorMessages.CityCannotBeNull);
-            }
-            if (city.Trim().Length == 0)
-            {
-                throw new ArgumentException(PatientErrorMessages.CityCannotBeEmpty);
-            }
-            if (state == null)
-            {
-                throw new ArgumentException(PatientErrorMessages.StateCannotBeNull);
-            }
-            if (state.Trim().Length == 0)
-            {
-                throw new ArgumentException(PatientErrorMessages.StateCannotBeEmpty);
-            }
-            if (zipcode == null)
-            {
-                throw new ArgumentException(PatientErrorMessages.ZipcodeCannotBeNull);
-            }
-            if (zipcode.Trim().Length == 0)
-            {
-                throw new ArgumentException(PatientErrorMessages.ZipcodeCannotBeEmpty);
-            }
-            if (phoneNumber == null)
-            {
-                throw new ArgumentException(PatientErrorMessages.PhoneNumberCannotBeNull);
-            }
+            if (city == null) throw new ArgumentException(PatientErrorMessages.CityCannotBeNull);
+            if (city.Trim().Length == 0) throw new ArgumentException(PatientErrorMessages.CityCannotBeEmpty);
+            if (state == null) throw new ArgumentException(PatientErrorMessages.StateCannotBeNull);
+            if (state.Trim().Length == 0) throw new ArgumentException(PatientErrorMessages.StateCannotBeEmpty);
+            if (zipcode == null) throw new ArgumentException(PatientErrorMessages.ZipcodeCannotBeNull);
+            if (zipcode.Trim().Length == 0) throw new ArgumentException(PatientErrorMessages.ZipcodeCannotBeEmpty);
+            if (phoneNumber == null) throw new ArgumentException(PatientErrorMessages.PhoneNumberCannotBeNull);
```

Comment:
This looks real messy, I don't know why you removed the braces but they need to be put back in.

Reply:
Again, Resharper... Has been fixed.